app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+    @created_events = @user.created_events
+
+    @future_events = @user.attended_events.where('event_date > ?', Date.today).order('event_date ASC')
+    @past_events = @user.attended_events.where('event_date <  ?', Date.today).order('event_date DESC')
```

Comment:
Since you have a `current_user` helper, `past` and `future` `scopes`, you should take advantage of them. for example you can have:
- [ ] `@created_events = current_user.create_events`
- [ ] `@past_events = current_user.attended_events.past`
- [ ] `@future_events = current_user.attended_events.future`